bread/layout/components/email_input.py

```diff
@@ -0,0 +1,15 @@
+from htmlgenerator import DIV
+
+from bread.layout.components.icon import Icon
+from bread.layout.components.text_input import TextInput
+
+
+class EmailInput(TextInput):
+    def __init__(self, *args, **kwargs):
```

Comment:
Nitpicking: the variable ```icon``` could be avoided by directly passing the ```DIV``` as argument to append. This can be controversial (therefore my label "nitpicking"), but I think it reduces mental and perceptional load.